spec/factories/requirements.rb

```diff
@@ -0,0 +1,11 @@
+FactoryGirl.define do
+  factory :requirement do
+    association :task
```

Comment:
I would prefer not to have the associations in the factory, but build them in the specs, as needed.